### DISCUSSION TITLE
update multiple records with multiple conditions (each record has its own condition parameter)

### DISCUSSION THREAD

**Discussion by jaydip-github** [prisma/prisma#15757] [Q&A] [answered]
can we **update multiple records with multiple conditions** (each record has its own condition parameter), is it possible?

**Comment by nurul3101** [comment, marked as answer]
Hi @jaydip-github 👋 
If you are referring to updating multiple records in a single query with multiple different conditions then it's not possible. You could achieve this in a single [transaction](https://www.prisma.io/docs/concepts/components/prisma-client/transactions#interactive-transactions-in-preview) though by invoking multiple update queries each with it's own condition.

**Reply by vladi-stevanovic** [reply]
👋 Hello @jaydip-github did you perhaps have a chance to check Nurul's recommendation? 
Let us know if you still have any questions, or feel free to "Mark as answer" if this discussion is now solved! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
